pynestml/utils/ast_utils.py

```diff
@@ -449,10 +451,31 @@ def create_state_block(cls, model: ASTModel):
         """
         # local import since otherwise circular dependency
         from pynestml.meta_model.ast_node_factory import ASTNodeFactory
-        if not model.get_internals_blocks():
-            state = ASTNodeFactory.create_ast_block_with_variables(True, False, False, list(),
+        if not model.get_state_blocks():
```

Comment:
Could you update the docs of this function?

Reply:
Done!